stle.css

```diff
@@ -448,3 +468,289 @@ header {
   color: #5b5957;
   background: #f4f1f0;
 }
+
+@media (min-width: 1180px) {
+  * {
+    margin: 0;
+    padding: 0;
+  }
```

Comment:
Please ensure that the media queries for the Desktop breakpoint value should be `min-width` property should be in the range of `992px` - `1100px`).

```css
// Doesn't pass the requirement (too small)
@media screen and (min-width: 990px) {
/*
.
.
*/
}

// Passes the requirement (proper size)
@media screen and (min-width: 1080px) { // please note that it doesn’t have to be exactly 1080px, any value between 992 and 1100 is ok.
/*
.
.
*/
}
```